LICENSE-3rdparty.csv

```diff
@@ -61,7 +61,7 @@ prometheus-client,PyPI,Apache-2.0,Copyright 2015 Brian Brazil
 prometheus-client,PyPI,Apache-2.0,Copyright 2015 The Prometheus Authors
 protobuf,PyPI,BSD-3-Clause,Copyright 2008 Google Inc.
 protobuf,PyPI,BSD-3-Clause,Copyright 2014 protobuf@googlegroups.com
-psutil,PyPI,BSD-3-Clause,"Copyright (c) 2009, Jay Loden, Dave Daeschler, Giampaolo Rodola"
+psutil,PyPI,BSD-3-Clause,"Copyright (c) 2009, Jay Loden, Dave Daeschler, Giampaolo Rodola'"
```

Comment:
If this is not intentional, let's change it back. Rest look good to me.

Reply:
no, this typo got fixed with 5.9.7 and it should be part of our license file https://github.com/DataDog/integrations-core/pull/16547/files#diff-7643b72018189103a0687e3fc5e39df19803006b523aab1e24c930d6ed1f5959R64